griptape/schemas/base_schema.py

```diff
@@ -186,3 +236,7 @@ def is_list_sequence(cls, field_type: type | _SpecialForm) -> bool:
                 return issubclass(field_type, Sequence)
         else:
             return False
+
+    @classmethod
+    def is_union_(cls, field_type: type) -> bool:
+        return field_type is Union or get_origin(field_type) is Union
```

Comment:
Can you please update this and `is_list_sequence` to be protected methods?

Reply:
how would you make these protected methods?

Reply:
Add a leading underscore. [More info](https://www.geeksforgeeks.org/access-modifiers-in-python-public-private-and-protected/).

Reply:
@collindutter nvm figured that out.